torchaudio/pipelines/_wav2vec2/impl.py

```diff
@@ -70,11 +88,13 @@ def get_model(self, *, dl_kwargs=None) -> Wav2Vec2Model:
             dl_kwargs (dictionary of keyword arguments): Passed to :func:`torch.hub.load_state_dict_from_url`.
         """
         model_type = self._params.pop("model_type", None)
+        normalize_waveform = self._params.pop("normalize_waveform", None)
         if model_type == "WavLM":
             model = wavlm_model(**self._params)
         else:
             model = wav2vec2_model(**self._params)
         model.load_state_dict(self._get_state_dict(dl_kwargs))
+        model = _Wav2Vec2Model(model, normalize_waveform)
```

Comment:
Let's keep the Wav2Vec2Model intact for cases that does not require the normalization.

```
if self._params.get("normalize_waveform", False):
    model = _Wav2Vec2Model(model)
```

and make `_Wav2Vec2Model` always perform `layer_norm` for now.